data/tutorials/language/1ms_00_modules.md

```diff
@@ -123,94 +119,118 @@ val list_sum_sq : int -> int = <fun>
 # let array_sum_sq m =
     Array.(init m Fun.id |> map (fun i -> i * i) |> fold_left ( + ) 0);;
 val array_sum_sq : int -> int = <fun>
-
 ```
 
-## Interfaces and Signatures
+## Interfaces and Implementations
+
+By default, anything defined in a module is accessible from other modules.
+Values, functions, types, or submodules, everything is public. This can be
+restricted. That allows distinguishing content provided to other modules from
+internal use content. What is internal is kept private and not available from
+other modules.
```

Comment:
```suggestion
By default, anything defined in a module is accessible from other modules, including
values, functions, types, or submodules. Everything is public. However, this can be
restricted, so distinguished content provided to other modules can be distinguished from
internal use content. Internal information is kept private and not available from
other modules.
```

Not sure about that second-to-last sentence. If I got it wrong, let's work on it togther.